src/TitleBindPieces/TitleBindPiecesContainer/TitleBindPiecesContainer.js

```diff
@@ -155,7 +154,7 @@ export const TitleBindPiecesContainer = () => {
         onSubmit={onSubmit}
         paneTitle={paneTitle}
         instanceId={title?.instanceId}
-        locations={holdingLocations}
+        titleId={titleId}
```

Comment:
what's the purpose of `titleId` prop if it's not used eventually? `(<TitleBindPieces>` - `<TitleBindPiecesCreateItemForm>`)